lib/serializer.ts

```diff
@@ -156,10 +158,10 @@ export class Serializer {
       } else if (typeName.match(/^Stream$/ig) !== null) {
         const objectType = typeof value;
         if (objectType !== "string" &&
-            objectType !== "function" &&
-            !(value instanceof ArrayBuffer) &&
-            !ArrayBuffer.isView(value) &&
-            !(typeof Blob === "function" && value instanceof Blob)) {
+          objectType !== "function" &&
```

Comment:
I'd prefer if this either were reindented or a blank line were inserted above the `throw` statement in order to make it clear where the condition ends and the body begins.

Reply:
I'll insert the blank line. I agree with you about the indentation, but the VS Code TypeScript formatter made it this way.